pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -497,12 +500,51 @@ func validatingHasNoSideEffects(webhooks []admissionregistration.ValidatingWebho
 	return true
 }
 
+// validatingWebhookAllowInvalidLabelValueInSelector returns true if all webhooksallow invalid label value in selector
+func validatingWebhookAllowInvalidLabelValueInSelector(webhooks []admissionregistration.ValidatingWebhook) bool {
+	for _, hook := range webhooks {
+		if hook.NamespaceSelector != nil {
+			allErrors := metav1validation.ValidateLabelSelector(hook.NamespaceSelector, false, field.NewPath("namespaceSelector"))
+			if len(allErrors) > 0 {
+				return false
+			}
+		}
+		if hook.ObjectSelector != nil {
+			allErrors := metav1validation.ValidateLabelSelector(hook.ObjectSelector, false, field.NewPath("objectSelector"))
+			if len(allErrors) > 0 {
+				return false
```

Comment:
this is backwards... if the existing selector failed validation, then we should return `true` because this webhook has an invalid label value. add a unit test that would have caught this